app/views/answers/_status.html.erb

```diff
@@ -2,8 +2,24 @@
 <!--
   Partial for handling the answer status (e.g. saving, error-saving, data-status)
 -->
-<span class="label label-info status" style="display:none;" data-status="saving"><%= _('Saving...') %></span>
-<span class="label label-warning status" style="display: none;" data-status="error-saving"></span>
+<%
+  helpdesk_email = Rails.configuration.x.organisation.helpdesk_email
+  email_subject = _('Plan writing error related to %{tool_name}') %{ :tool_name => tool_name }
+%>
+<span class="label label-warning status" style="display:none;" data-status="saving">
+  <%= _("Saving...Please do not refresh or go to another page!") %>
+  <br><br>
+  <%= sanitize(_("If you stuck on this message, please save your plan to a separate file and contact us at %{helpdesk_email} .") % {
```

Comment:
I think "If you are stuck on this message" might be confusing since it doesn't let them know what is supposed to happen (e.g. how long is 'stuck'?). 

Suggest we either change the messaging (maybe ask the PMs)

Reply:
> I think "If you are stuck on this message" might be confusing since it doesn't let them know what is supposed to happen (e.g. how long is 'stuck'?).
> 
> Suggest we change the messaging (maybe ask the PMs)

How about "If this message doesn't disappear in 1 minute?"  @mariapraetzellis how do you feel?

Reply:
@mariapraetzellis @dsisu : one more UI message suggestion is needed. For anyone sticking on the 'Saving...' message because of a system problem, we need them to remember not to fresh the page and to contact the helpdesk immediately. 

Instead of saying `If you are stuck on this message, please save your plan to a separate file and contact us at %{helpdesk_email} `, any suggestions you have?